docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -177,7 +177,8 @@ command to start the rest api:
 
 .. code-block:: console
 
-  rest_api --stream-url tcp://127.0.0.1:4004
+  rest_api --connect tcp://127.0.0.1:4004
```

Comment:
I just merged a PR that conflicts with this (it drops the endpoint since the default works). Please rebase.